src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,23 @@
+package duke.task;
+
+import duke.task.Task;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+
+  private LocalDate date;
+
+  public Event(String description, boolean status, LocalDate date) {
+    super(description, status);
+    this.date = date;
+  }
+
+  public Event(String description, LocalDate date) {
+    this(description, false, date);
+  }
+
+  public String toString() {
+    return String.format("[E][%s] %s (at: %s)", super.isMarked(), super.getDescription(), DateTimeFormatter.ofPattern("MMM d yyyy"));
+  }
+}
```

Comment:
Perhaps add header comments for the class and public functions. I noticed the same issue for classes extending Task and Command?